pkg/gcs/ast/parseTarget.go

```diff
@@ -46,6 +46,30 @@ func parseTarget(p *Parser) (parseFn, error) {
 					return nil, err
 				}
 				r.Pos.R = amt
+			case "type":
+				item, err := p.acceptSeqReturnLast(itemAssign, itemIdentifier)
+				if err != nil {
+					return nil, err
+				}
+				params, err := p.acceptOptionalTargetParams()
+				if err != nil {
+					return nil, err
+				}
+				err = enemy.ConfigureTarget(&r, item.Val, params)
+				if err != nil {
+					return nil, err
+				}
+				p.res.Settings.DamageMode = true
```

Comment:
With this current parsing logic, I think target configuration will be order dependent? Not sure if this is something we want or how we'd want to account for this if not (some struct union logic I guess would do the trick?)

IE: the following will create a dummy with resistance `0.1` in all categories
```
target lvl=100 resist=0.2 type=dummy;
```

whereas the following will result in resistances set to `0.2`
```
target lvl=100 type=dummy resist=0.2;
```

Reply:
I am for this idea. Even though it's a breaking change, it's only "broken" for people who abused syntax.